test/parallel/test-debugger-repeat-last.js

```diff
@@ -10,7 +10,7 @@ const fixture = path.join(
 
 const args = [
   'debug',
-  `--port=${common.PORT}`,
+  '--port=0',
```

Comment:
This won't work. `debug` cannot accept port 0 on the command line. It requires a port between 1024 and 65535.